### PR TITLE
Translated a few features from operations

### PR DESCRIPTION
**Pull request by nrbernier**

### CODE REVIEW

File: hcga/features/link_analysis_hits.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        .. [2] Jon Kleinberg,
+           Authoritative sources in a hyperlinked environment
+           Journal of the ACM 46 (5): 604-32, 1999.
+           doi:10.1145/324133.324140.
+           http://www.cs.cornell.edu/home/kleinber/auth.pdf.
+
+        """
+        def hits(graph):
+            h,a = nx.hits(graph ,max_iter=1000)
+            return np.asarray(list(h.values()))
```
Comment (arnaudon) [inline hcga/features/link_analysis_hits.py:77]: I think that just a list works, right?

File: hcga/features/node_clique_number.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        Notes
+        -----
+        Clique number calculations using networkx:
+            `Networkx_clique <https://networkx.github.io/documentation/stable/reference/algorithms/clique.html>`_
+        """        
+                
+        
+        self.add_feature(
+            "clique sizes",
+            lambda graph: np.asarray(list(clique.node_clique_number(utils.ensure_connected(graph)).values())),
```
Comment (arnaudon) [inline hcga/features/node_clique_number.py:56]: same here

File: hcga/features/link_analysis_hits.py (hunk 1 above)
Comment (nrbernier) [reply hcga/features/link_analysis_hits.py:77]: I copy-pasted the previous code. Yes, I agree that some recasts look superfluous.

**Comment by arnaudon** [comment]
run black also, so you get the spaces correct: h, a = and (graph, max_iter=1000)

**Comment by nrbernier** [comment]
Removed asarray and ran black.

**Pull request merged**